### PR TITLE
Dutch translation

### PR DESCRIPTION
**Pull request by TheGroundZero**

### CODE REVIEW

File: include/translations/NL.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "systemTitle": "Alarmsysteem",
+
+    "armed_away": "Actief - Afwezig",
+    "armed_home": "Actief - Thuis",
+    "armed_night": "Actief - Nacht",
+    "disarmed": "Uitgeschakeld",
+    "pending": "Alarm in afwachting",
+    "triggered": "!!! Alarm !!!",
+    "arming": "Activeren...",
+    "requested": "Wachten op een antwoord",
```
Comment (TheGroundZero) [inline include/translations/NL.h]: This will be too long for a 20 char display.
"Wachten op antwoord" fits.

File: include/translations/NL.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    "enterCode": "Voer code in",
+    "invalidCodeHeader": "Foutieve code",
+    "invalidCodeMessage": "Code geweigerd",
+
+    "connecting": "Connecteren met server",
```
Comment (TheGroundZero) [inline include/translations/NL.h]: Also too long for 20 char display
"Verbinden met server" fits.

**Review by TheGroundZero — COMMENTED** [review]
I may have rushed a bit on the commit.
Added comments to 2 lines that exceeded 20 chars and proposed a shorter line.

**Comment by paviro** [comment]
Thank you will check in a moment!

**Pull request merged**